docs/tutorials/log.rst

```diff
@@ -4,6 +4,11 @@
 Log of major tutorial updates
 #############################
 
+2025-12-16
+==========
+
+Updated notebook tutorials 103.4, 303.2, and 306.2.
```

Comment:
Just a bit more detail here about the change, so maybe add "to work with the new cutout service".